src/warc2zim/content_rewriting/html.py

```diff
@@ -161,3 +139,50 @@ def handle_pi(self, data: str):
 
     def unknown_decl(self, data: str):
         self.handle_decl(data)
+
+    def process_attr(
+        self,
+        attr: tuple[str, str | None],
```

Comment:
What is expected value? Since we don't expand

Reply:
I don't get your question, sorry

Reply:
Nevermind, I got it.

Reply:
Now it is expanded ^^